added stickermule sponsor

Pegysus opened the pull request:
- Added Stickermule logo and link to their website in a Sponsor section
- Added information for other sponsors in sponsors.json but still have yet to implement their links on the section

Pegysus committed:
rebase with main

Pegysus committed:
center aligned image, added sponsor text, and fixed a typo

rileysw committed:
updated sponsor assets, refactored sponsor section

rileysw commented:
This is a temporary solution since the deadline to publish the Sticker Mule logo is by Wednesday morning. The temporary design implementation (as well as the data in sponsors.json) was approved by Parsa.

I think both @Pegysus and me were confused about using the image relative paths from sponsors.json since it wouldn't load correctly into the image component.

taesungh commented on src/views/home/sections/Sponsors/Sponsors.tsx:
```diff
@@ -0,0 +1,46 @@
+
+function SponsorItem(props: any) {
+	return (
+		<Link href={props.data.url}>
```
Suggestion: `next/link` should be used only for internal routing, not to link to external sites.

taesungh commented on src/views/home/sections/Sponsors/Sponsors.module.scss:
```diff
@@ -0,0 +1,14 @@
+	height: 200px;
+}
+
+.smallTierImage:hover {
+	transform: scale(1.1);
+	transition: all 0.3s ease-in-out;
+}
```
Suggestion: specifiers can be nested with SASS.

taesungh committed:
Fix divergence during rebase

- As of 84d5d4a, `MuseumRoom` was moved to be included in `MainLayout`

taesungh committed:
Clean up props, styles, imports for Sponsors

- Unify text centering for `h2` elements on Home page
- Define props for `SponsorItem`
- Use regular anchor instead of `next/link` for sponsor links
- Fix hover transform transition

taesungh reviewed (commented):
Few more things to address, I'm working on most of them.

taesungh committed:
Use white background for Sticker Mule logo

- The Sticker Mule Brand Guidelines suggest placing their logo on their
  specific background colors which are their version of brown or white
  - To avoid conflicting with our `$dark-brown`, use a white background

taesungh commented on src/views/home/sections/Sponsors/Sponsors.tsx:
```diff
@@ -0,0 +1,46 @@
+
+import stickerMule from "assets/sponsors/sticker_mule.svg";
+
+function SponsorItem(props: any) {
```
`SponsorItemProps` added in 0511137.

rileysw committed:
added email link to sponsors section

taesungh commented on src/views/home/sections/Sponsors/Sponsors.tsx:
```diff
@@ -0,0 +1,56 @@
+		<section id="sponsors" className="container">
+			<h2>Sponsors</h2>
+			<p className={styles.sponsorInfo}>
+				Interested in sponsoring HackUCI 2023? Check out our information above
+				to learn more about our event! For more information, please email us
```
Suggestion: use of word "information" seems a little redundant/conflicting. Perhaps say "For additional inquiries" in second sentence.

Pegysus committed:
Edited sponsor info wording

samderanova commented on src/views/home/sections/Sponsors/Sponsors.tsx:
```diff
@@ -0,0 +1,56 @@
+	return (
+		<section id="sponsors" className="container">
+			<h2>Sponsors</h2>
+			<p className={styles.sponsorInfo}>
+				Interested in sponsoring HackUCI 2023? Check out our information above
+				to learn more about our event! For additional inquiries, please email us
+				at&nbsp;
+				<a href="mailto: hack@uci.edu">hack@uci.edu</a>.
+			</p>
```
Nitpick: We should center the paragraph element and change the color of the link to something other than blue (perhaps same orange as the link in the footer) for consistency.

samderanova commented on src/views/home/sections/Sponsors/Sponsors.tsx:
```diff
@@ -0,0 +1,56 @@
+				<a href="mailto: hack@uci.edu">hack@uci.edu</a>.
+			</p>
+			<Row className="my-4">
+				<Col className={styles.column}>
+					<SponsorItem {...stickerMule} src={logo_stickerMule} />
+				</Col>
```
Awesome animation!

samderanova reviewed (changes requested):
Thanks for working on this! Just have one small suggestion.

Pegysus committed:
Center sponsor text and change email link color

taesungh commented:
Please fix the formatting/linting errors rather than dismissing them.

taesungh reviewed (approved):
Thanks for starting this. Temporary solution works for now; we'll clean up more later.
